app/views/owner/planets/index.html.erb

```diff
@@ -10,3 +10,11 @@
   <% end %>
 </ul>
 <%= link_to "Add a planet", new_planet_path %>
+<% if policy(@planet).update? %>
```

Comment:
I think this is the same as the previous request, but you can delete lines 13-20 here

Reply:
Other than that, every thing looks great!